process/management/commands/api_loader.py

```diff
@@ -30,23 +29,16 @@ def callback(client_state, channel, method, properties, input_message):
     collection_id = input_message["collection_id"]
     url = input_message["url"]
     path = input_message.get("path")
-    errors = input_message.get("errors")
 
     collection = Collection.objects.get(pk=collection_id)
     if collection.deleted_at:
         ack(client_state, channel, method.delivery_tag)
         return
 
     with transaction.atomic():
-        if errors:
-            create_note(collection, CollectionNote.Level.ERROR, f"Couldn't download {url}", data=json.loads(errors))
-            collection_file = None
-        else:
-            filename = os.path.join(settings.KINGFISHER_COLLECT_FILES_STORE, path)
-            collection_file = create_collection_file(collection, filename=filename, url=url)
-
-    # FileError items from Kingfisher Collect are not processed further.
-    if collection_file:
+        filename = os.path.join(settings.KINGFISHER_COLLECT_FILES_STORE, path)
+        collection_file = create_collection_file(collection, filename=filename, url=url)
+
         message = {"collection_id": collection_id, "collection_file_id": collection_file.pk}
         publish(client_state, channel, message, routing_key)
```

Comment:
We don't want the message publication inside `transaction.atomic`. If the DB transaction fails, there should be no message published.